spec/emoji_data_spec.rb

```diff
@@ -13,13 +13,19 @@
 
   describe ".all_doublebyte" do
     it "should return an array of all 21 known emoji chars with doublebyte encoding" do
-      EmojiData.all_doublebyte.count.should eq(21)
+      EmojiData.all_doublebyte.count.should eq(71)
```

Comment:
Where do these extra doublecodepoint chars come from? Are they additional flags that Apple added support for?